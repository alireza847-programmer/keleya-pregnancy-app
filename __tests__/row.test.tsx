import React from 'react';
import renderer from 'react-test-renderer';
import {test, expect, jest} from '@jest/globals';
import Row from 'components/uiElements/row';

test('renders correctly', () => {
  const tree = renderer.create(<Row />).toJSON();
  expect(tree).toMatchSnapshot();
});
