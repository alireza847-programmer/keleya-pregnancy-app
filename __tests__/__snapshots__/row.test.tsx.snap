// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "justifyContent": "space-between",
        "marginTop": 0,
        "width": 600,
      },
      {
        "flexDirection": "row",
      },
      undefined,
    ]
  }
/>
`;
